autoscheduler/scheduler/tests/scheduling_tests.py

```diff
@@ -0,0 +1,520 @@
+""" Tests all of the models functions """
```

Comment:
I'm not sure the best method to do it, but the fact that GitHub doesn't load this files diff by default since it's so large makes me think we should def split this up into different files.